locale/en/knowledge/intermediate/how-to-log.md

```diff
@@ -85,6 +87,6 @@ Because of the file transport we set up, winston also logged the warning to 'som
 
 Note that winston's file logger formats the logs differently for file logging (JSON in this case) than it does for the console transport.
 
-Winston also supports logging to Riak, CouchDB, MongoDB and even [Loggly](http://loggly.com).  The `logger.warn` call we used before also put the same message into each database, according to the options we gave to each transport.
+Winston also supports logging to Riak, CouchDB, MongoDB (need install winston-riak、winston-mongo and winston-couchdb to support Riak、CouchDB、MongoDB) and even [Loggly](http://loggly.com).  The `logger.warn` call we used before also put the same message into each database, according to the options we gave to each transport.
```

Comment:
I'd shorten it to something like: 

> Winston also supports logging to Riak, CouchDB, MongoDB and [many other transports](https://github.com/winstonjs/winston/blob/master/docs/transports.md). The `logger.warn` …

As there are a lot more than the ones already showed above, and with the link everybody can find out if their specific logging scenario is supported.